src/pipecat/services/gladia/stt.py

```diff
@@ -204,7 +204,7 @@ def __init__(
         api_key: str,
         region: Literal["us-west", "eu-west"] | None = None,
         url: str = "https://api.gladia.io/v2/live",
-        confidence: float = 0.5,
+        confidence: float = 0.3,
```

Comment:
Communicated with Gladia since it seems to provide low confidence even for clear audio.